www/translations/locale-en.json

```diff
@@ -26,8 +26,9 @@
     "draft-04-invalid": "Sample draft-04 schema and <strong>invalid</strong> document"
   },
 
-  "WARNING_V5_UNOFFICIAL": "The v5-unofficial schema version, provided by Ajv to try experimental features, was previously erroneously named \"draft-05\".<br />This is, and will not be,  an official JSON Schema version and <em>should not be used</em>.  Please see the schema dropdown for the latest official version.",
-
+  "WARNING_V5_UNOFFICIAL": "This schema version, v5-unofficial, was a prototype for a draft version that was eventually scrapped. Previously, JSON Schema Lint offered this prototype as \"draft-05\", as it was originally meant to supersede draft-04.<br />This is not, and will never be, an official JSON Schema version and <em>should not be used for new projects</em>.  Please see the schema dropdown for the latest official version.",
+  "WARNING_EXPERIMENTAL": "This is an experimental meta-schema used for testing new features. Currently uses draft-06 with the <a href='https://epoberezkin.github.io/ajv/#data-reference'>$data keyword</a> enabled,<br /> along with additional <a href='https://github.com/epoberezkin/ajv-keywords'>custom keywords</a> provided by Ajv. Features may be added or removed from Experimental at any time.",
```

Comment:
As with v5-unofficial, can we be very explicit about advising against using this for anything but academic interest?